terraform/schema.yaml

```diff
@@ -772,6 +779,144 @@ variables:
+    description: "The domain-wide administration port to configure a secure WebLogic domain"
+    required: true
+
+  ms_administration_port:
+    visible:
+      and:
+        - ${orm_create_mode}
+        - ${configure_wls_ports}
+        - ${configure_secure_mode}
+    type: integer
+    default: 9004
+    minimum: 1024
+    maximum: 65535
+    multipleOf: 1
+    title: "Administration Port for Managed Servers"
+    description: "The administration port for managed servers to configure a secure WebLogic domain"
+    required: true
+
+  keystore_password_id:
+    visible:
+      and:
+        - ${orm_create_mode}
+        - ${configure_secure_mode}
+    type: "oci:kms:secret:id"
+    minLength: 1
+    maxLength: 1024
+    required: true
+    title: "Validated Secret for Keystore password"
+    description: "The secret that contains the keystore password. To create secrets, see <a target=\"_blank\" href=\"https://docs.oracle.com/en/cloud/paas/weblogic-cloud/user/you-begin-oracle-weblogic-cloud.html#GUID-0517A019-4A89-40ED-A568-DD2193733F8B\"> Create Secrets for Passwords</a>."
+    dependsOn:
+      compartmentId: ${wls_admin_secret_compartment_id}
+
+  root_ca_id:
+    visible:
+      and:
+        - ${orm_create_mode}
+        - ${configure_secure_mode}
+    type: string
+    pattern: ^ocid1.certificateauthority.*$
+    required: true
+    title: "Existing Root Certificate Authority ID"
+    description: "The OCID of the existing root certificate authority to issue the certificates"
+
+  cert_compartment_id:
+    visible:
+      and:
+        - ${orm_create_mode}
+        - ${configure_secure_mode}
+        - ${create_policies}
+    type: oci:identity:compartment:id
+    required: false
+    title: "Certificate Compartment"
+    description: "The compartment where you want to create the certificate"
+    default: ${network_compartment_id}
+
+  thread_pool_limit:
+    visible:
+      and:
+        - ${orm_create_mode}
+        - and:
+            - ${configure_secure_mode}
+    type: string
+    default: 65536
+    title: "Throttle the thread pool"
+    description: "Shared Capacity For Work Managers"
+    required: true
+
+  wls_primary_admin_user:
+    visible:
+      and:
+        - ${orm_create_mode}
+        - ${configure_secure_mode}
+    type: string
+    title: "WebLogic Server Admin User Name"
+    description: "The name of the primary administrator in the WebLogic Server domain. The value should be between 8 and 128 characters long and alphanumeric, and can contain underscore (_) and hyphen(-) special characters, and should not be system, admin, administrator, or weblogic."
+    pattern: "^(?!weblogic$|administrator$)[a-zA-Z][a-zA-Z0-9_-]{7,127}$"
+    default: "wls_user"
+    minLength: 8
+    maxLength: 128
+    required: true
+
+  wls_secondary_admin_user:
+    visible:
+      and:
+        - ${orm_create_mode}
+        - ${configure_secure_mode}
+    type: string
+    title: "WebLogic Server Admin User Name"
```

Comment:
I wonder why wls_primary_admin_user and wls_secondary_admin_user use the same title: 

title: "WebLogic Server Admin User Name"

Reply:
Updated